src/OrbitGl/Track.cpp

```diff
@@ -159,7 +161,7 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
   float button_offset = layout.GetCollapseButtonOffset();
   float toggle_y_pos = pos_[1] + half_label_height;
   Vec2 toggle_pos = Vec2(tab_x0 + button_offset, toggle_y_pos);
-  collapse_toggle_->SetPos(toggle_pos);
+  collapse_toggle_->SetPos(toggle_pos[0], toggle_pos[1]);
```

Comment:
Why this change?

Reply:
Before this change, `TriangleToggle` had its own `SetPos` method which received a Vec2, but the new base class only receives x/y. Forgot this in the last commit, but this shouldn't be here, I'll move it to the previous commit. Thanks for finding this!